src/configs/elm.rs

```diff
@@ -1,22 +1,17 @@
-use crate::config::{ModuleConfig, RootModuleConfig, SegmentConfig};
+use crate::config::{ModuleConfig, RootModuleConfig};
 
-use ansi_term::{Color, Style};
 use starship_module_config_derive::ModuleConfig;
 
 #[derive(Clone, ModuleConfig)]
 pub struct ElmConfig<'a> {
-    pub symbol: SegmentConfig<'a>,
-    pub version: SegmentConfig<'a>,
-    pub style: Style,
+    pub format: &'a str,
```

Comment:
@jletey Thank you for your contribution. I thought I have mentioned it in other PRs of yours, Adding `symbol` and `style` option is recommended as per discussion in https://github.com/starship/starship/discussions/1116#discussioncomment-6579 .
```suggestion
    pub format: &'a str,
    pub symbol: &'a str,
    pub style: &'a str,
```